coordinator/gscoordinator/utils.py

```diff
@@ -157,9 +164,8 @@
 LLVM4JNI_USER_OUT_DIR_BASE = "user-llvm4jni-output"
 PROCESSOR_MAIN_CLASS = "com.alibaba.graphscope.annotation.Main"
 JAVA_CODEGNE_OUTPUT_PREFIX = "gs-ffi"
-GRAPE_PROCESSOR_JAR = os.path.join(
-    GRAPHSCOPE_HOME, "lib", "grape-runtime-0.16.0-shaded.jar"
-)
+GRAPE_PROCESSOR_JAR = glob.glob(GRAPHSCOPE_HOME + "/lib/grape-runtime*.jar")[0]
```

Comment:
This [0] seems dangerous...
Could you handle the exception incase the jar is not there? So we won't crash the coordinator.

Reply:
Thanks for the suggestion, I have changed to 
```python 
ANALYTICAL_ENGINE_JAVA_RUNTIME_JAR = os.path.join(
    ANALYTICAL_ENGINE_JAVA_HOME,
    "lib",
    "grape-runtime-{}-shaded.jar".format(__version__),
)
```
where the `__version__` is imported from version.py